tests/test_lint.py

```diff
@@ -266,3 +267,29 @@ def test_expired_metric():
 
     assert len(nits) == 1
     assert set(["EXPIRED"]) == set(v.check_name for v in nits)
+
+
+def test_expires_too_far_in_the_future():
+    """Test that a misformatted `expires` field generates errors"""
```

Comment:
It isn't misformatted as much as it is just very far in the future.

Also maybe put in a note that this test is expected to start failing in 2098 : D

Reply:
Yup, for got to change this docstring, oops.

> Also maybe put in a note that this test is expected to start failing in 2098 : D

LOL, sure.